### PR TITLE
docs(v1.5.7): add v1.5.7 release notes to CHANGELOG

### PR DESCRIPTION
**Pull request by ewanharris**
## Description

Updates the changelog for the upcoming v1.5.7 release.

Diff between main and 1.5.6 is viewable [here](https://github.com/openfga/openfga/compare/v1.5.6...main), I'm not familiar enough to determine if the following are consequential enough for includiong

- https://github.com/openfga/openfga/pull/1789 - This is mostly impacting tests I think?

## Review Checklist
- [x] I have clicked on ["allow edits by maintainers"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [ ] I have added documentation for new/changed functionality in this PR or in a PR to [openfga.dev](https://github.com/openfga/openfga.dev) [Provide a link to any relevant PRs in the references section above]
- [x] The correct base branch is being used, if not `main`
- [ ] I have added tests to validate that the change in functionality is working as expected

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,10 +8,18 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
...
 ## [Unreleased]
 
+## [1.5.7] - 2024-07-25
+
 ### Added
 
 * Support requesting a different consistency option per request in `Check`, `Expand`, `ListObjects`, `ListUsers`, and `Read` [#1764](https://github.com/openfga/openfga/pull/1764)
+  * This is currently experimental and needs to be enabled by configuring `OPENFGA_EXPERIMENTALS=enable-consistency-params` or passing `--experimentals enable-consistency-params` to `openfga run`.
   * When `HIGHER_CONSISTENCY` is requested, OpenFGA will skip the check resolver cache. For storage implementors it is recommended to skip any caching and perform a stronger read if `HIGHER_CONSISTENCY` is requested. This can be accessed in the `Consistency` options provided to the relevant methods of the storage interface.
+* Publish images to GitHub Container Registry [#1775](https://github.com/openfga/openfga/pull/1775) - Thanks @JAORMX!
```
Comment (jpadilla) [inline CHANGELOG.md:18]:
```suggestion
* Start publishing images to `ghcr.io/openfga/openfga` as alternative to DockerHub [#1775](https://github.com/openfga/openfga/pull/1775) - Thanks @JAORMX!
```

**Review by jpadilla — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,10 +8,19 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
...
 
+## [1.5.7] - 2024-07-25
+
 ### Added
 
 * Support requesting a different consistency option per request in `Check`, `Expand`, `ListObjects`, `ListUsers`, and `Read` [#1764](https://github.com/openfga/openfga/pull/1764)
+  * This is currently experimental and needs to be enabled by configuring `OPENFGA_EXPERIMENTALS=enable-consistency-params` or passing `--experimentals enable-consistency-params` to `openfga run`.
   * When `HIGHER_CONSISTENCY` is requested, OpenFGA will skip the check resolver cache. For storage implementors it is recommended to skip any caching and perform a stronger read if `HIGHER_CONSISTENCY` is requested. This can be accessed in the `Consistency` options provided to the relevant methods of the storage interface.
+* Start publishing images to `ghcr.io/openfga/openfga` as alternative to DockerHub [#1775](https://github.com/openfga/openfga/pull/1775) - Thanks @JAORMX!
+* Performance improvements for parent child relations in Check [#1765](https://github.com/openfga/openfga/pull/1765)
```
Comment (miparnisari) [inline CHANGELOG.md:19]: Please also add https://github.com/openfga/openfga/pull/1786:

> Performance improvement in Check: computed relations don't consume from the resolution depth quota, doesn't trigger additional goroutines, and doesn't get cached.

**Comment by miparnisari** [comment]
> https://github.com/openfga/openfga/pull/1789 - This is mostly impacting tests I think?

Technically it also affects production code, but it's such a rare edge case (server starts and shuts down immediately for whatever reason) that it's not worth mentioning.

File: CHANGELOG.md (hunk 2 above)
Comment (ewanharris) [reply CHANGELOG.md:19]: Added in #6a93ae97e171dc17c1670a4f648508c1c77b736d

**Review by miparnisari — APPROVED** [review]
